backend/shared/shared/audit_log/README.md

```diff
@@ -52,6 +52,6 @@ audit_logging.log(
 ```
 
 Based on:
-- [apartment-application-service audit logging](https://github.com/City-of-Helsinki/apartment-application-service/tree/develop/audit_log)
+- [apartment-application-service audit logging](https://github.com/City-of-Helsinki/apartment-application-service/tree/main/audit_log)
```

Comment:
This is not yjdh monorepo's link but I checked it out and going to the old link here actually redirected to the main branch so it is ok to change it for this repo also